src/components/navbar/NavBar.tsx

```diff
@@ -46,6 +47,11 @@ const NavBar = () => {
               <p className={$navbar.linkTitle}>Settings</p>
             </Link>
             <br />
+            <Link href="/about" className={$navbar.link}>
+              <Info className={$navbar.icon} />
```

Comment:
Please be sure you are adding the property types to the appropriate .css file - and fix some of the "any" type errors.